src/traces/contour/attributes.js

```diff
@@ -42,6 +42,18 @@ module.exports = extendFlat({
     yhoverformat: axisHoverFormat('y'),
     zhoverformat: axisHoverFormat('z', 1),
     hovertemplate: heatmapAttrs.hovertemplate,
+    texttemplate: extendFlat({}, heatmapAttrs.texttemplate, {
+        description: [
+            'For this trace it only has an effect if `coloring` is set to *heatmap*.',
```

Comment:
Why this restriction? Not a big deal, probably won't get a lot of use in contour maps anyway. But the only issue I can think of is what autocolor to give the text - for `coloring='fill'` it should work pretty well with the same logic as heatmap, and for other `coloring` values we'd just contrast with the `plot_bgcolor`. No?

Reply:
I agree we may try to add this option for other `coloring` values in another PR.
But that would require extra work as those use different `plot` paths.

Reply:
A contour plot with a `coloring` other than *heatmap* has different plotting path. So enabling this feature requires extra work. So it could potentially be added later on the road.